### PR TITLE
Add neko command

### PR DESCRIPTION
**Pull request by pedrofracassi**
Fetches a random neko girl (lewd if the channel is NSFW) from https://nekos.life/ and displays it.

**Usage:** `s!neko`

### CODE REVIEW

**Comment by pedrofracassi** [comment]
@xDoges oh

File: src/commands/anime/neko.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    super(client)
+    this.name = 'neko'
+    this.aliases = ['nekogirl']
+  }
+
+  async run (message) {
+    message.channel.startTyping()
+
+    // Send a lewd neko if the channel is NSFW
+    let endpoint = message.channel.nsfw ? 'lewd' : 'neko'
```
Comment (davipatury) [inline src/commands/anime/neko.js]: `const`

Comment (pedrofracassi) [reply src/commands/anime/neko.js]: Oh, yeah. Sure.

File: src/commands/anime/neko.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    message.channel.startTyping()
+
+    // Send a lewd neko if the channel is NSFW
+    const endpoint = message.channel.nsfw ? 'lewd' : 'neko'
+
+    const { body } = await snekfetch.get(nekoAPI + endpoint)
+    message.channel.send(
+      new SwitchbladeEmbed(message.author)
+        .setImage(body.url)
+        .setDescription(`Here's your neko!`)
```
Comment (davipatury) [inline src/commands/anime/neko.js]: Simples quote

**Pull request merged**